tests/testthat/_snaps/parse-news-items/NEWS.md

```diff
@@ -1,6 +1,6 @@
 <!-- NEWS.md is maintained by https://cynkra.github.io/fledge, do not edit -->
 
-# fledge 0.0.1 (2023-01-23)
+# fledge 0.0.1
```

Comment:
Is this intentional?

Reply:
yes! otherwise it varies every day, it was a mistake from earlier.

Reply:
Can we use a fixed date?